PF/Assignment1/CollegeList.java

```diff
@@ -0,0 +1,70 @@
+package prg2;
+
+//has list of college
+public class CollegeList {
+	static int[] count = new int[5];
+	static String[][] studentsList = new String[5][4];
+
+	int MLV(String name) {
+
+		if (count[0] < 4) {// check for seat if available or not
+			studentsList[0][(count[0])] = name;
+			count[0]++;
+			System.out.println(name + "was alloted");
+			return 1;// if alloted
+		} else {
+			System.out.println("Seats Full");
+			return 0;// if not alloted
+		}
+	}
+
+	int MIT(String name) {
```

Comment:
ACTION: Instead of create replication of code for each college, create constants for college name and create a single method to allotment.